components/form/TaskForm.tsx

```diff
@@ -65,63 +62,47 @@ export function CreateTaskForm({
 
     return (
         <>
-            <div style={{ padding: "20px" }}>
-                <FormControl isInvalid={!taskFields.name} isRequired>
-                    <FormLabel>Task Name</FormLabel>
-                    <Input
-                        placeholder="Task Name"
-                        onChange={(event) =>
-                            setTaskFields({
-                                ...taskFields,
-                                name: event.target.value,
-                            })
-                        }
-                    />
-                    <FormErrorMessage>
-                        Task name cannot be empty.
-                    </FormErrorMessage>
-                </FormControl>
-                <FormControl>
-                    <FormLabel>Description</FormLabel>
-                    <Input
-                        placeholder="Description"
-                        onChange={(event) =>
-                            setTaskFields({
-                                ...taskFields,
-                                description: event.target.value,
-                            })
-                        }
-                    />
-                </FormControl>
-                <FormControl py="0.5rem">
-                    <Flex flexDirection={"row"} alignItems={"baseline"}>
-                        <FormLabel>Billable</FormLabel>
-                        <Checkbox
-                            isChecked={taskFields.billable}
-                            onChange={(event) =>
-                                setTaskFields({
-                                    ...taskFields,
-                                    billable: event.target.checked,
-                                })
-                            }
-                        />
-                    </Flex>
-                </FormControl>
-            </div>
-            <div style={{ textAlign: "right", padding: "20px" }}>
-                <Button colorScheme="blue" mr={3} onClick={onSubmit}>
-                    Submit
-                </Button>
-                <Button colorScheme="gray" onClick={onCancel}>
-                    Cancel
-                </Button>
-            </div>
-            {errorMessage && (
-                <>
-                    <ErrorAlert />
-                    <Box>{errorMessage}</Box>
-                </>
-            )}
+            <FormContainer>
```

Comment:
I'd prefer the containers to be arranged as 
```typescript
<FormContainer>
    <Stuff />
</FormContainer>
```

instead of
```typescript
<>
    <FormContainer>
        <Stuff />
    </FormContainer>
    <FormContainer>
        <Stuff />
    </FormContainer>
</>
```

If you want another container for the fields you can just define something like `FormFieldsContainer` separately and just nest it.